index.html

```diff
@@ -59,9 +59,9 @@ <h4 class="project-tittle pt1">Project name goes here</h4>
                             </ul>
                         </div>
                         <button class="project-button">See this project
-                            <span><img class="b1" src="./images/ic_arrow_right.png" alt=""></span>
+                            <span><img class="b1" src="./images/ic_arrow_right.png" alt="arrow-button"></span>
```

Comment:
[ OPTIONAL] 
- [ ] Kindly consider putting your name instead of the sample name provided in Figma. `instead of Charlie ... you can put your name`
![image](https://user-images.githubusercontent.com/97100091/193935008-12c2bbfc-b216-49a6-9666-2c4ff4b380a9.png)

Reply:
- [ ] You can also consider personalizing the `About` and `Get started` sections instead of `Lorem ipsum.....`
`hint: You can refer to other templates`